src/types.ts

```diff
@@ -32,8 +34,22 @@ export interface ZAuctionInstances {
   [registrarAddress: string]: zAuction.Instance;
 }
 
-export interface RouteUriToInstance {
-  [key: string]: ZAuctionInstances;
+export interface NetworksToAddresses {
+  [address: string]: Maybe<AddressesToTokenApiInfo>;
+}
+
+export interface AddressesToTokenApiInfo {
```

Comment:
Nit: What are these addresses? As a developer here I know it's a token address, but from the interface it's not clear.
Also could we be more verbose on what `TokenApiInfo` is? This is info specifically for coin gecko so maybe we could call this `CoinGeckoApiParams`?